src/elements/embed/Callout.tsx

```diff
@@ -0,0 +1,217 @@
+import { css } from "@emotion/react";
+import { neutral, space, text } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+import React, { useEffect, useState } from "react";
+import { Error } from "../../editorial-source-components/Error";
+import { Label } from "../../editorial-source-components/Label";
+import { FieldLayoutVertical } from "../../editorial-source-components/VerticalFieldLayout";
+import type { FieldNameToValueMap } from "../../plugin/helpers/fieldView";
+import { unescapeHtml } from "../helpers/html";
+import { EmbedTestId } from "./EmbedForm";
+import type { createEmbedFields } from "./EmbedSpec";
+
+type Props = {
+  fieldValues: FieldNameToValueMap<ReturnType<typeof createEmbedFields>>;
+};
+
+type Callout = {
+  id: string;
+  name: string;
+  fields: TagFields;
+};
+
+type TagFields = {
+  tagName: string;
+  callout: string;
+  description: string;
+  formUrl: string;
+};
+
+const calloutStyles = css`
+  ${textSans.small({ fontWeight: "regular", lineHeight: "loose" })}
+  font-family: "Guardian Agate Sans";
+  a {
+    color: ${text.anchorPrimary};
+  }
+  code {
+    font-family: monospace;
+    background-color: ${neutral[86]};
+    border-radius: ${space[1]}px;
+    padding: 1px 4px;
+  }
+  table {
+    width: 100%;
+    border-collapse: collapse;
+    border: 1px solid ${neutral[86]};
+    font-size: 14px;
+  }
+  th,
+  tr,
+  td {
+    border: 1px solid ${neutral[86]};
+    padding: ${space[1]}px;
+    line-height: 14px;
+    vertical-align: top;
+  }
+  th {
+    text-align: left;
+  }
+  p,
+  p:first-child {
+    margin-top: 0px;
+    margin-bottom: 0px;
+  }
+  ul {
+    margin-bottom: 0px;
+  }
+`;
+
+const marginBottom = css`
+  margin-bottom: ${space[2]}px !important;
+`;
+
+const getCampaigns = (tag: string) => {
+  return fetch("https://targeting.gutools.co.uk/api/campaigns")
```

Comment:
Is there a more specific endpoint we can use for this?

Reply:
(We did memoise this one in the end while pairing)